src/components/Personalization/handlers/injectCreateProposition.js

```diff
@@ -69,11 +78,23 @@ export default ({ preprocess, isPageWideSurface }) => {
         return PROPOSITION_SCOPE_TYPE;
       },
       getItems() {
-        return items.map(item => createItem(item, this));
+        const first = [];
+        const last = [];
+        items.forEach(rawItem => {
+          const item = createItem(rawItem, this);
+          Array.prototype.push.call(
+            item.getSchemaType() !== DOM_ACTION_CLICK ? first : last,
+            item
+          );
+        });
+        return [...first, ...last];
```

Comment:
As you can see, the `getItems` method was modified to ensure any `click` proposition items are at the end of the list.  This goes hand in hand with `executeSequentially`.  During authoring, a user can define a `click` to track.  Those click proposition items can appear anywhere within that array depending on when the user made the change.  The problem with that is that we need all the DOM elements to be rendered prior to the click handler.  That way every click handler will be sure to have the element it relies on already rendered to the DOM. 

This wasn't an issue for target, because click items always appeared within a second proposition for the activity.  Not the case in AJO.